eng/targets/Imports.BeforeArcade.targets

```diff
@@ -28,4 +28,14 @@
     <UseAppHost>false</UseAppHost>
   </PropertyGroup>
 
+  <!-- TODO: Remove when roslyn updates to Arcade 9.
+       Arcade provides the IsTestUtilityProject handling natively there,
+       setting IsShipping=false in those cases, and excluding them from the build
+       when in source build or vertical build modes. Note, at a later date tests will
+       be built by default in vertical mode. -->
+  <PropertyGroup Condition="'$(IsTestUtilityProject)' == 'true'">
+    <IsShipping>false</IsShipping>
```

Comment:
@ViktorHofer This won't actually work in this repo. The arcade computation of IsShipping is calculated very early (via the SDK's BeforeCommonTargets hook, so this is too late).

I'm going to revert the "IsShipping" changes but leave IstestUtilityProject. When roslyn moves to 9.0 Arcade, the IsShipping properties in those projects can be removed.